sidecar/src/agent_remote_config.rs

```diff
@@ -22,7 +23,12 @@ fn path_for_endpoint(endpoint: &Endpoint) -> CString {
     // We need a stable hash so that the outcome is independent of the process
     let mut hasher = ZwoHasher::default();
     endpoint.url.authority().unwrap().hash(&mut hasher);
-    CString::new(format!("/libdatadog-agent-config-{}", hasher.finish())).unwrap()
+    CString::new(format!(
+        "/ddcfg-{}-{}",
```

Comment:
macos has a length limit of 30 chars, this fits it.

Reply:
Maybe add a comment in the code explaining that?

Reply:
Yes :-D